FlowCrypt/Models/Realm Models/KeyInfoRealmObject.swift

```diff
@@ -19,23 +19,22 @@ enum KeyInfoError: Error {
     case missingPrimaryFingerprint
 }
 
-final class KeyInfo: Object {
+final class KeyInfoRealmObject: Object {
+    @Persisted(primaryKey: true) var primaryFingerprint = ""
+    @Persisted var `private`: String = ""
+    @Persisted var `public`: String = ""
+    @Persisted var passphrase: String?
+    @Persisted var source: String = ""
+    @Persisted var user: UserRealmObject!
+
     var primaryLongid: String {
         allLongids[0]
     }
 
-    @objc dynamic var `private`: String = ""
-    @objc dynamic var `public`: String = ""
-
     let allFingerprints = List<String>()
```

Comment:
Looks like I did mistake here because of this:

> To declare properties of generic types LinkingObjects, List, and RealmProperty, use let. Generic properties cannot be represented in the Objective‑C runtime, which Realm Database uses for dynamic dispatch of dynamic properties.

So this properties must be persisted too. To avoid such mistake in future we must use `@Persisted`

Reply:
This is the reason why UI tests failed and this is good to UI test

Reply:
Yup. You can only do limited testing with unit tests. They won't keep you safe.